addons/stock/tests/test_inventory.py

```diff
@@ -19,13 +19,13 @@ def setUpClass(cls):
         cls.product1 = cls.env['product.product'].create({
             'name': 'Product A',
             'type': 'product',
-            'categ_id': cls.env.ref('product.product_category_all').id,
+            'categ_id': cls.env.ref('product.cat_services').id,
```

Comment:
For all the `stock` related tests, you should use a category who reflect better a storable product.
It is not so meaningful for now but it would better reflect real use case.
I guess furniture should be more appropriate